src/test/java/org/opensearch/security/authtoken/jwt/JwtVendorTest.java

```diff
@@ -99,7 +99,9 @@ public void testCreateJwtWithRoleSecurityMode() throws Exception {
         Settings settings = Settings.builder()
             .put("signing_key", "abc123")
             .put("encryption_key", claimsEncryptionKey)
-            .put(ConfigConstants.EXTENSIONS_BWC_PLUGIN_MODE, true)
+             // CS-SUPPRESS-SINGLE: RegexpSingleline get Extensions Settings
```

Comment:
not sure that this is right solution. in case of an extensions feature I drag behind